### ISSUE TITLE
java.lang.IllegalArgumentException: Buffering capacity 2097152 exceeded

### ISSUE THREAD

**Issue by webshark** [blazegraph/database#232]
Getting this error while running the wikidata runUpdater.sh

java.lang.IllegalArgumentException: Buffering capacity 2097152 exceeded

I don't see where I can set this config to a higher number

**Comment by thompsonbry** [comment]
Stack trace?

**Comment by webshark** [comment]
I just have the log from the debug mode:

12:46:53.179 [update 4] DEBUG o.w.q.r.t.w.WikibaseRepository - Fetching rdf from https://www.wikidata.org/wiki/Special:EntityData/Q103933897.ttl?flavor=dump&nocache=1656679613179
12:46:53.186 [update 7] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 95 ms
12:46:53.187 [update 7] DEBUG org.wikidata.query.rdf.tool.Updater - Processing data for Q111804028@1651920458@20220601124651|1703581390
12:46:53.187 [update 7] DEBUG o.w.q.r.t.w.WikibaseRepository - Fetching rdf from https://www.wikidata.org/wiki/Special:EntityData/Q111804028.ttl?flavor=dump&nocache=1656679613187
12:46:53.214 [update 5] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 101 ms
12:46:53.214 [update 5] DEBUG org.wikidata.query.rdf.tool.Updater - Processing data for Q94562706@1651920485@20220601124654|1703581419
12:46:53.214 [update 5] DEBUG o.w.q.r.t.w.WikibaseRepository - Fetching rdf from https://www.wikidata.org/wiki/Special:EntityData/Q94562706.ttl?flavor=dump&nocache=1656679613214
12:46:53.228 [update 1] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 327 ms
12:46:53.228 [update 0] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 106 ms
12:46:53.228 [update 1] DEBUG org.wikidata.query.rdf.tool.Updater - Processing data for Q5896227@1651920427@20220601124649|1703581359
12:46:53.228 [update 1] DEBUG o.w.q.r.t.w.WikibaseRepository - Fetching rdf from https://www.wikidata.org/wiki/Special:EntityData/Q5896227.ttl?flavor=dump&nocache=1656679613228
12:46:53.228 [update 0] DEBUG org.wikidata.query.rdf.tool.Updater - Processing data for Q50093816@1651920497@20220601124655|1703581428
12:46:53.228 [update 0] DEBUG o.w.q.r.t.w.WikibaseRepository - Fetching rdf from https://www.wikidata.org/wiki/Special:EntityData/Q50093816.ttl?flavor=dump&nocache=1656679613228
12:46:53.230 [update 6] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 110 ms
12:46:53.231 [update 6] DEBUG org.wikidata.query.rdf.tool.Updater - Processing data for Q103933898@1651920437@20220601124650|1703581371
12:46:53.231 [update 6] DEBUG o.w.q.r.t.w.WikibaseRepository - Fetching rdf from https://www.wikidata.org/wiki/Special:EntityData/Q103933898.ttl?flavor=dump&nocache=1656679613231
12:46:53.269 [update 3] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 216 ms
12:46:53.271 [update 9] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 142 ms
12:46:53.279 [update 2] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 202 ms
12:46:53.281 [update 8] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 108 ms
12:46:53.298 [update 7] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 111 ms
12:46:53.300 [update 4] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 121 ms
12:46:53.322 [update 1] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 94 ms
12:46:53.332 [update 6] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 101 ms
12:46:53.355 [update 0] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 127 ms
12:46:53.395 [update 5] DEBUG o.w.q.r.t.w.WikibaseRepository - Done in 180 ms
12:46:53.424 [main] DEBUG o.w.query.rdf.tool.rdf.RdfRepository - Processing 80 IDs and 15199 statements
12:46:53.467 [main] DEBUG o.w.q.rdf.tool.rdf.client.RdfClient - Completed in 42 ms
12:46:53.606 [main] DEBUG o.w.query.rdf.tool.rdf.RdfRepository - Sending query 4331986 bytes
12:47:03.497 [main] INFO  o.w.query.rdf.tool.HttpClientUtils - HTTP request failed: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Buffering capacity 2097152 exceeded, attempt 1, will retry
12:47:08.990 [main] INFO  o.w.query.rdf.tool.HttpClientUtils - HTTP request failed: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Buffering capacity 2097152 exceeded, attempt 2, will retry
12:47:16.425 [main] INFO  o.w.query.rdf.tool.HttpClientUtils - HTTP request failed: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Buffering capacity 2097152 exceeded, attempt 3, will retry

**Comment by thompsonbry** [comment]
Ok. That is in the wikidata integration.  You want to check with them.
Bryan

**Comment by webshark** [comment]
Yeah been trying to - can’t get anyone to respond. I guess I’ll just keep trying
